### PR TITLE
New Mix Tasks Lesson (en)

### PR DESCRIPTION
**Pull request by pjhampton**
I could do with some early feedback here. I stole this idea from you @sivsushruth - maybe you could look over this pull request as well and offer some feedback?

screen shot 2016-04-27 at 18 29 16

### CODE REVIEW

File: lessons/basics/mix-tasks.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+```
+
+## Custom Mix Task
+
+Now with lib create a new directory and file **hello/lib/mix/tasks/hello.ex**. Within this file, let's insert these 7 lines of Elixir.
+
+```elixir
+defmodule Mix.Tasks.Hello do
+  use Mix.Task
+
```
Comment (sivsushruth) [inline lessons/basics/mix-tasks.md]: For tasks to show in `mix help` we need to add something like 
```
@shortdoc "Simply calls the Hello.say/0 function"
```
So when we do `mix help` we get 
```
mix hello                 # Simply calls the Hello.say/0 function
```

Comment (pjhampton) [inline lessons/basics/mix-tasks.md]: Ahh, I didn't know this 😄  I'll add this in

**Comment by Koziolek** [comment]
One more request/question. Is it possible in Elixir to make custom mix task as dependency? If yes, add short example, please.

**Comment by pjhampton** [comment]
@Koziolek once this is packaged up and deployed, anybody who adds this as a dependency to their project

```
defp deps do
  [
    {:exrm, "~> 0.18.1"}
  ]
end
```

Once you run 

```
$ mix deps.get
```

You get the command line task

```
$ mix release
```

Do you think this needs to be made explicit?

**Comment by doomspork** [comment]
This is great @pjhampton 👍

**Comment by doomspork** [comment]
@pjhampton any changes you want to make to this or can we merge? 😀

**Comment by pjhampton** [comment]
Yeah :shipit: @doomspork. If I come up with anything new, I will open a new PR

**Comment by doomspork** [comment]
:boom:  Awesome work @pjhampton!

**Comment by doomspork** [comment]
@pjhampton can you update the various Issues to include this new lesson so the translators are aware?

**Comment by pjhampton** [comment]
@doomspork done 💥

**Pull request merged**